### PR TITLE
Simplify db vars

### PR DESCRIPTION
**Pull request by davekiss**

### CODE REVIEW

**Review by dylanjha — APPROVED** [review]

File: .env.example [hunk 1]
```diff
@@ -1,10 +1,30 @@
...
 MUX_TOKEN_ID=
 MUX_TOKEN_SECRET=
+
+# Optionally verify incoming webhooks are from Mux
+# https://docs.mux.com/guides/video/verify-webhook-signatures
+
 MUX_WEBHOOK_SECRET=
+
+# Easily generate a new value for this secret here
+# https://generate-secret.vercel.app/
```
Comment (steven-tey) [inline .env.example:28]: can we move this before the optional `MUX_WEBHOOK_SECRET` var

File: .env.example [hunk 2]
```diff
@@ -1,10 +1,30 @@
-DATABASE_URL="mysql://root@127.0.0.1:3309/video-course-starter-kit"
+# Set this value to the canonical URL of your site
+
 NEXTAUTH_URL="http://localhost:3000"
```
Comment (steven-tey) [inline .env.example:3]: Can we move this to the bottom and note that this is only required for local development? (no need to set it in prod)

**Review by steven-tey — COMMENTED** [review]
left 2 small comments, other than that LGTM!

File: .env.example (hunk 2 above)
Comment (steven-tey) [reply .env.example:3]: the `NEXTAUTH_URL` I mean

File: .env.example (hunk 1 above)
Comment (steven-tey) [reply .env.example:28]: the `NEXTAUTH_SECRET` var I mean

**Pull request merged**